releases/8.4/languages/tr.php

```diff
@@ -0,0 +1,57 @@
+    'new_standard_functions' => 'Yeni fonksiyonlar: <code>http_get_last_response_headers()</code>, <code>http_clear_last_response_headers()</code> ve <code>fpow()</code>.',
+    'new_xml_functions' => 'Yeni metotlar: <code>XMLReader::fromStream()</code>, <code>XMLReader::fromUri()</code>, <code>XMLReader::fromString()</code>, <code>XMLWriter::toStream()</code>, <code>XMLWriter::toUri()</code> ve <code>XMLWriter::toMemory()</code>.',
+    'new_grapheme_function' => 'Yeni <code>grapheme_str_split()</code> fonksiyonu.',
+
+    'bc_title' => 'Kullanımdan Kaldırmalar ve Geriye Dönük Uyumluluk Kırılmaları',
+    'bc_pecl' => 'IMAP, OCI8, PDO_OCI ve pspell uzantıları ayrılmış ve PECL’e taşınmıştır.',
+    'bc_nullable_parameter_types' => 'Varsayılan olarak nullable olan parametre türleri artık desteklenmiyor.',
+    'bc_classname' => 'Bir sınıf adı olarak <code>_</code> kullanımı artık desteklenmiyor.',
+    'bc_zero_raised_to_negative_number' => 'Sıfırın negatif bir sayıya üssü artık desteklenmemektedir.',
+    'bc_gmp' => '<code>GMP</code> sınıfı artık final olarak tanımlanmıştır.',
+    'bc_round' => '<code>round()</code> fonksiyonuna geçersiz bir mod gönderildiğinde artık <code>ValueError</code> fırlatır.',
+    'bc_typed_constants' => '<code>date</code>, <code>intl</code>, <code>pdo</code>, <code>reflection</code>, <code>spl</code>, <code>sqlite</code> ve <code>xmlreader</code> uzantılarındaki sınıf sabitleri artık türlendirilmiş durumda.',
+    'bc_mysqli_constants' => '<code>MYSQLI_SET_CHARSET_DIR</code>, <code>MYSQLI_STMT_ATTR_PREFETCH_ROWS</code>, <code>MYSQLI_CURSOR_TYPE_FOR_UPDATE</code> gibi sabitler artık kaldırılmıştır.',
+    'bc_mysqli_functions' => '<code>mysqli_ping()</code>, <code>mysqli_kill()</code>, <code>mysqli_refresh()</code> fonksiyonları, <code>mysqli::ping()</code>, <code>mysqli::kill()</code>, <code>mysqli::refresh()</code> metotları ve <code>MYSQLI_REFRESH_*</code> sabitleri artık kullanımdan kaldırıldı.',
+    'bc_standard' => '<code>stream_bucket_make_writeable()</code> ve <code>stream_bucket_new()</code> artık <code>stdClass</code> yerine bir <code>StreamBucket</code> objesi döndürüyor.',
+    'bc_core' => '<a href="/manual/en/migration84.incompatible.php#migration84.incompatible.core.exit"><code>exit()</code> davranış değişikliği</a>.',
+    'bc_warnings' => '<code>E_STRICT</code> sabiti artık kullanımdan kaldırılmıştır.',
+
+    'footer_title' => 'Daha iyi performans, daha temiz sözdizimi, gelişmiş tür güvenliği.',
+    'footer_description' => '<p>PHP 8.4 kaynak dosyalarını indirmek için <a href="/downloads">indirme sayfasını</a> ziyaret edin. Windows için çalıştırılabilir dosyaları <a href="https://windows.php.net/download">PHP for Windows</a> sitesinden indirebilirsiniz. Tüm değişikliklerin listesi <a href="/ChangeLog-8.php#PHP_8_4">ChangeLog</a> içinde kayıtlıdır.</p>
+        <p>Detaylı bilgi için <a href="/manual/en/migration84.php">göç rehberine</a> göz atabilirsiniz.</p>',
```

Comment:
```suggestion
        <p>Detaylı bilgi için <a href="/manual/tr/migration84.php">göç rehberine</a> göz atabilirsiniz.</p>',
```

Hey!

Would you like to provide links `/manual/tr/*` instead of `/manual/en/*` here and elsewhere?

Reply:
@saundefined Since my whole life has been spent with English documents, I’ve gotten so used to /en links that they didn’t even seem odd to me. Great catch. Thank you!